src/openrct2/interface/window.h

```diff
@@ -497,6 +497,7 @@ enum {
 #define WC_TOP_TOOLBAR__WIDX_WATER                      9
 #define WC_TOP_TOOLBAR__WIDX_SCENERY                    10
 #define WC_TOP_TOOLBAR__WIDX_PATH                       11
+#define WC_TOP_TOOLBAR__WIDX_CLEAR_SCENERY              17
```

Comment:
This needs an accompanying `validate_global_widx(WC_TOP_TOOLBAR, WIDX_CLEAR_SCENERY)` in `windows/top_toolbar.c`